mlkem/verify.c

```diff
@@ -48,23 +48,29 @@ void cmov(uint8_t *r, const uint8_t *x, size_t len, uint8_t b) {
 }
 
 /*************************************************
- * Name:        cmov_int16
+ * Note:
  *
- * Description: Copy input v to *r if b is 1, don't modify *r if b is 0.
- *              Requires b to be in {0,1};
- *              Runs in constant time.
+ * Constant-time implementation. Relies on basic
+ * properties of bitwise ^ or and &, namely that:
  *
- * Arguments:   int16_t *r:       pointer to output int16_t
- *              int16_t v:        input int16_t
- *              uint8_t b:        Condition bit; has to be in {0,1}
+ * x ^ 0 == x
```

Comment:
I feel that this is overly detailed -- can you please remove it?

Reply:
Done. The explanatory comments in the body are sufficient to explain how it works.